### PR TITLE
Fix Makefile.local-late test

### PR DESCRIPTION
**Pull request by SkySkimmer**
Instead of outputting to stdout in the makefile and looking at the log
(which includes random make verbosities), we output to a file.

### CODE REVIEW

File: test-suite/coq-makefile/local-late-extension/run.sh [hunk 1]
```diff
@@ -2,14 +2,13 @@
 
 . ../template/path-init.sh
 
+rm -rf _test
+mkdir _test
+find . -maxdepth 1 -not -name . -not -name _test -exec cp -r '{}' -t _test ';'
+cd _test
+
```
Comment (JasonGross) [inline test-suite/coq-makefile/local-late-extension/run.sh:9]: I'm a bit confused why we need this

File: test-suite/coq-makefile/local-late-extension/run.sh [hunk 2]
```diff
@@ -2,14 +2,13 @@
...
-make --no-print-directory -j1 printvo 2>&1 \
-    | grep -v 'make[^:]*: warning: -jN forced in submake: disabling jobserver mode.' \
-    | grep -v 'coq_makefile -Q . Top foo.v -o Makefile' \
-    | grep -v 'make[^:]*: Entering directory' \
-    | grep -v 'make[^:]*: Leaving directory' \
-    | grep -v 'exit 1' \
-           > make-printvo.log
-diff -u make-printvo.log.expected make-printvo.log || exit $?
+rm -f .*.d ./*.d
+make --no-print-directory -j1 printvo
```
Comment (JasonGross) [inline test-suite/coq-makefile/local-late-extension/run.sh:13]: The `rm` is no longer necessary now that we're not taking the entire log; it was there only to normalize the log with regard to emitting info about generating .d files.
```suggestion
make --no-print-directory -j1 printvo
```

Comment (JasonGross) [reply test-suite/coq-makefile/local-late-extension/run.sh:13]:
```suggestion
make printvo
```
We actually also don't care about `-j1` and `--no-print-directory`.  They don't hurt anything, but they're no longer needed now that we're outputting to a file.

File: test-suite/coq-makefile/local-late-extension/Makefile.local-late [hunk 3]
```diff
@@ -1,4 +1,4 @@
 MYVOFILES := $(VOFILES)
 
 printvo::
-	@echo '$(MYVOFILES)'
+	@echo '$(MYVOFILES)' > $@
```
Comment (JasonGross) [inline test-suite/coq-makefile/local-late-extension/Makefile.local-late:4]: Maybe get rid of the `@` so it's easier to debug if needed?
```suggestion
	echo '$(MYVOFILES)' > $@
```

**Review by JasonGross — APPROVED** [review]
Thanks for this improvement!  I've left some suggestions for further improvements.

File: test-suite/coq-makefile/local-late-extension/Makefile.local-late (hunk 3 above)
Comment (SkySkimmer) [reply test-suite/coq-makefile/local-late-extension/Makefile.local-late:4]: it changes nothing though

File: test-suite/coq-makefile/local-late-extension/run.sh (hunk 1 above)
Comment (SkySkimmer) [reply test-suite/coq-makefile/local-late-extension/run.sh:9]: it was that or add `printvo` to gitignore somewhere
I feel this way is nicer

Comment (JasonGross) [reply test-suite/coq-makefile/local-late-extension/run.sh:9]: Alternatively we could pick a filename that's already ignored, like make it output to `$@.log` or something.

Comment (JasonGross) [reply test-suite/coq-makefile/local-late-extension/run.sh:9]: But also it doesn't seem like `_test` will be ignored?

Comment (SkySkimmer) [reply test-suite/coq-makefile/local-late-extension/run.sh:9]: https://github.com/coq/coq/blob/master/test-suite/coq-makefile/.gitignore

Comment (JasonGross) [reply test-suite/coq-makefile/local-late-extension/run.sh:9]: Ah, I see.  And you think this is nicer than making it output to `printvo.log` (given that .log files are already .gitignored)?

Comment (JasonGross) [reply test-suite/coq-makefile/local-late-extension/run.sh:9]: Sure.  I've also already approved.  Should I assign as well?

**Review by JasonGross — APPROVED** [review]

**Comment by ppedrot** [comment]
Given that @JasonGross approved the change and this is quite annoying, I'll assign myself and quickly merge.

**Review by ppedrot — APPROVED** [review]

**Pull request merged**